src/view/treeNodes/treeNode.ts

```diff
@@ -66,6 +67,36 @@ export abstract class TreeNode implements vscode.Disposable {
 
 	updateFromCheckboxChanged(_newState: vscode.TreeItemCheckboxState): void { }
 
+	static processCheckboxUpdates(checkboxUpdates: vscode.TreeCheckboxChangeEvent<TreeNode>) {
```

Comment:
Probably not the ideal place for this. I wanted somewhere easily accessible for both `prChangesTreeDataProvider.ts` and `prsTreeDataProvider.ts`. Where would you recommend I put it?

Reply:
There isn't an existing place that this fits nicely. Let's put it in a new `treeUtils.ts` file next to `treeNode.ts`.